shared/util/feature-flags.native.js

```diff
@@ -8,9 +8,6 @@ const ff: FeatureFlags = {
   impTeamChatEnabled: true,
   newPeopleTab: __DEV__,
   plansEnabled: false,
-  tabGitEnabled: true,
-  tabPeopleEnabled: false,
```

Comment:
false on mobile, but we already determine which tabs to show in a platform-specific manner